### ISSUE TITLE
[UI] Access to the gaming room. Can look around. But can't move

### ISSUE THREAD

**Issue by Lancelot20** [itsamewayu/OLFallenDoll#140]
**Describe the bug**
Closed Beta. I access to the gaming room already. But everything I can do is look around in the elevator. M, E ( experiment), Tab, Space,... I can use all of them. However, game never explains to me how can I move my character. 
I don't know if it is bug or not. I can't move, can't open the gate to go to other rooms. Just stuck in the elevator until now.

**Desktop (please complete the following information):**
 - OS: Window 10 64 bit
 - CPU: i5 10400F
 - VGA: RTX3060

**Additional context**
If it isn't bug. I think you need a tutorial or at least a control button list in the game's option. It will help the player to know how to use the game's control.

**Comment by alfi301** [comment]
Same Problem here, I do not even see a character
![Screenshot (157)](https://user-images.githubusercontent.com/95278116/144080650-d2db8903-e6ed-4140-b50b-11fde0e70f3f.png)

**Comment by salsabeard** [comment]
I've run into this several times now.  It seems to be an issue related to loading into the server.  As you'll notice, you're the only player listed.  The first time I got around this by leaving and joining a few times.  However, it kept failing after that.  But this last time, I let it sit for about 10-15 minutes and it finally spawned in my character as well as the other players.

**Comment by CanterburyFails** [comment]
Encountering the same issue, just an empty room, player character is not spawning either. No way to open doors. Can move mouse. If you go into the character customization menu there is no mouse cursor and then game eventually freezes and requires hard reset of PC to recover (Alt+F4 does not exit the app). Does game require other con-current players to play?

**Comment by jinjin525** [comment]
Duplicate of #101
